includes/Connector.class.php

```diff
@@ -7,6 +7,7 @@ class AC_Connector {
 	public $url;
 	public $api_key;
 	public $output = "json";
+	private $timeout = NULL;

```

Comment:
Let's default `timeout` to 30 seconds by using a `const` and assigning it by default to the `timeout` class property.